runtimes/braid/src/lib.rs

```diff
@@ -742,6 +742,8 @@ impl pallet_chain_space::Config for Runtime {
 parameter_types! {
 	pub const MaxDigestsPerBatch: u16 = 1_000;
 	pub const MaxRemoveEntries: u16 = 1_000;
+	pub const MaxSelectiveDataKeyLength: u32 = 100;
```

Comment:
Can we keep the length to 64? I guess it should be enough.

Reply:
Updated.